src/node/recursiveModuleCompile.js

```diff
@@ -50,6 +50,8 @@ function recursiveModuleCompileToSingleFile(outputFile, includes, options) {
     return include;
   });
 
+  options.rootModules = resolvedIncludes;
```

Comment:
My initial reaction was to agree with @pflannery: the better solution is metadata and it is probably better to do that in a separate PR.  Then #1859